docs/metrics.md

```diff
@@ -66,6 +66,9 @@ The current available memory of the VM containers based on the rss. Type: Gauge.
 ### kubevirt_vm_container_free_memory_bytes_based_on_working_set_bytes
 The current available memory of the VM containers based on the working set. Type: Gauge.
 
+### kubevirt_vm_created_total
+Amount of VMs created, broken down by namespace. Type: Counter.
```

Comment:
As @fabiand pointed out, you might want to elaborate on this.

Reply:
updated